cmd/sealer/cmd/alpha/mount.go

```diff
@@ -0,0 +1,119 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/containers/storage"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/sealerio/sealer/common"
+	"github.com/sealerio/sealer/pkg/define/options"
+	"github.com/sealerio/sealer/pkg/imageengine"
+	"github.com/sealerio/sealer/pkg/imageengine/buildah"
+)
+
+var (
+	path    string
+	imageID string
+)
+
+var longMountCmdDescription = `
+mount the cluster image to '/var/lib/sealer/data/overlay2' the directory and check whether the contents of the build image and rootfs are consistent in advance
```

Comment:
Can we only mount to this path? Is it possible to specify the mount path?

Reply:
> Can we only mount to this path? Is it possible to specify the mount path?

@Starnop Yes, mount to the default path; users don't need to care about the mount path, similar to the image file in containerd